src/NHibernate/Impl/SessionFactoryImpl.cs

```diff
@@ -462,7 +462,7 @@ private IQueryCache BuildQueryCache(string queryCacheName)
 			if (caches.TryGetValue(cacheKey, out var cache)) 
 				return cache;
 
-			cache = CacheFactory.CreateCache(strategy, GetCache(cacheRegion));
+			cache = CacheFactory.CreateCache(strategy, GetCache(cacheRegion), settings);
```

Comment:
The equivalent change was also done on `UpdateTimestampsCache` creation in the session factory. I have reverted it since the change had to be dropped from that `UpdateTimestampsCache` class.